service/route53/internal/customizations/doc.go

```diff
@@ -0,0 +1,41 @@
+/*
+Package customizations provides customizations for the Amazon Route53 API client.
+
+This package provides support for following customizations
+    Process Response Middleware: used for custom error deserializing
```

Comment:
lint seems to complain (on some platforms) about this style. We switched to doing `// ` prefix instead of `/*` to not have lint complain.